dotnet/samples/Demos/Hosting/Agent/ChatWithAgent.AppHost/appsettings.json

```
@@ -5,5 +5,13 @@
       "Microsoft.AspNetCore": "Warning",
       "Aspire.Hosting.Dcp": "Warning"
     }
-  }
+  },
+  "AIServices": {
+    "AzureOpenAIChat": {
+      "DeploymentName": "gpt-4o-mini",
+      "ModelName": "gpt-4o-mini",
+      "ModelVersion": "2024-07-18"
```

Should we stick the model here or add proper guidance to the Readme.md.

```suggestion
      "DeploymentName": "gpt-4o-mini",
      "ModelName": "gpt-4o-mini",
      "ModelVersion": "2024-07-18"
```

---

I plan to add section to readme describing how to use the configuration. The values in the config are good defaults.